### DISCUSSION TITLE
How to create a workout

### DISCUSSION THREAD

**Discussion by mfrutos** [matin/garth#70] [Q&A] [answered]
I'm looking for am example on how to create a workout

I can get my workouts:
```
garth.connectapi("/workout-service/workouts", method="GET",)
```

I can get details of one of them by passing the ID:
```
garth.connectapi("/workout-service/workout/575567355", method="GET",)
```

But if a try to do a POST to `/workout-service/workout/`, I only get 4xx errors. Can anybody help me here?

**Comment by matin** [comment, marked as answer]
The best way to create a workout is to upload a FIT file
